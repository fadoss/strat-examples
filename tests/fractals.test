params/fractals

select MAIN .
srew [1] a a a a a using map .
srew a a a a a using map .
srew [1] a a a a a using map2 .
srew [1] a a a a a using map3 .
srew [1] a a a a a using map4 .
srew [1] a a a a a using map5 .

select FRACTALS-STRAT .
srew (0.0, 0.0) >> (1.0, 0.0) using expandTimes(3) .
