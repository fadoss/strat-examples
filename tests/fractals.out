==========================================
srewrite [1] in MAIN : a a a a a using map .

Solution 1
rewrites: 10
result NeList{Letters0}: c c c c c
==========================================
srewrite in MAIN : a a a a a using map .

Solution 1
rewrites: 10
result NeList{Letters0}: c c c c c

No more solutions.
rewrites: 10
==========================================
srewrite [1] in MAIN : a a a a a using map2 .

Solution 1
rewrites: 10
result NeList{Letters0}: c c c c c
==========================================
srewrite [1] in MAIN : a a a a a using map3 .

Solution 1
rewrites: 11
result NeList{Letters0}: c c c c c
==========================================
srewrite [1] in MAIN : a a a a a using map4 .

Solution 1
rewrites: 5
result NeList{Letters0}: c c c c c
==========================================
srewrite [1] in MAIN : a a a a a using map5 .

Solution 1
rewrites: 5
result NeList{Letters0}: c c c c c
==========================================
srewrite in FRACTALS-STRAT : (0.0, 0.0) >> (1.0, 0.0) using expandTimes(3) .

No solution.
rewrites: 0
Bye.
